### PR TITLE
Chinese Localization

### PR DESCRIPTION
**Pull request by Cubik65536**
Chinese (Simplified) Translation for Minecraft-Console-Client

### CODE REVIEW

File: MinecraftClient/Resources/lang/zh-CHS.ini [hunk 1]
```diff
@@ -0,0 +1,501 @@
...
+cmd.inventory.creative_fail=请求创造性操作失败
+cmd.inventory.need_creative=你必须在创造模式
+cmd.inventory.container_not_found=找不到容器，请使用显式ID重试
+cmd.inventory.close=关闭背包 #{0}
+cmd.inventory.close_fail=关闭背包失败 #{0}
+cmd.inventory.not_exist=库存 #{0} do not exist
+cmd.inventory.inventory=库存
+cmd.inventory.inventories=存货
+cmd.inventory.hotbar=您选择的物品栏是 {0}
+cmd.inventory.damage=损坏
```
Comment (ReinforceZwei) [inline MinecraftClient/Resources/lang/zh-CHS.ini:271]: The damage here mean weapons damage (武器傷害值)

File: MinecraftClient/Resources/lang/zh-CHS.ini [hunk 2]
```diff
@@ -0,0 +1,501 @@
...
+cmd.list.players=玩家列表: {0}
+
+# Log
+cmd.log.desc=将文本记录到控制台.
+
+# Look
+cmd.look.desc=查看方向或坐标.
+cmd.look.unknown=未知方向 '{0}'
+cmd.look.at=查看相对偏差: {0} 间距: {1}
+cmd.look.block=Looking at {0}
```
Comment (ReinforceZwei) [inline MinecraftClient/Resources/lang/zh-CHS.ini:303]: Missing translation here.

File: MinecraftClient/Resources/lang/zh-CHS.ini [hunk 3]
```diff
@@ -0,0 +1,501 @@
...
+# Inventory
+cmd.inventory.desc=存储类型命令
+cmd.inventory.creative_done=请求 {0} x{1} 在插槽 #{2}
+cmd.inventory.creative_delete=请求清除插槽 #{0}
+cmd.inventory.creative_fail=请求创造性操作失败
+cmd.inventory.need_creative=你必须在创造模式
+cmd.inventory.container_not_found=找不到容器，请使用显式ID重试
+cmd.inventory.close=关闭背包 #{0}
+cmd.inventory.close_fail=关闭背包失败 #{0}
+cmd.inventory.not_exist=库存 #{0} do not exist
```
Comment (ReinforceZwei) [inline MinecraftClient/Resources/lang/zh-CHS.ini:267]: The message after `#` should be translated as well.

File: MinecraftClient/Resources/lang/zh-CHS.ini [hunk 4]
```diff
@@ -0,0 +1,501 @@
...
+bot.autoCraft.help.reload=重新加载配置文件.
+bot.autoCraft.help.resetcfg=将默认示例配置写入默认位置。
+bot.autoCraft.help.start=开始制作. 用法: /autocraft start <配方名称> 
+bot.autoCraft.help.stop=停止当前正在进行的合成过程
+bot.autoCraft.help.help=获取命令描述. 用法: /autocraft help <命令名>
+bot.autoCraft.loaded=已成功加载
+bot.autoCraft.start=自动合成启动中: {0}
+bot.autoCraft.start_fail=无法启动自动合成。检查你的可用制作材料 {0}
+bot.autoCraft.table_not_found=找不到合成列表
+bot.autoCraft.close_inventory=库存 #{0} was closed by AutoCraft
```
Comment (ReinforceZwei) [inline MinecraftClient/Resources/lang/zh-CHS.ini:381]: The message after `#` should be translated as well.

File: MinecraftClient/Resources/lang/zh-CHS.ini [hunk 5]
```diff
@@ -0,0 +1,501 @@
...
+# Terrain & Movements
+extra.terrainandmovement_enabled=地形和移动现在已启用。
+extra.terrainandmovement_disabled=§c该MC版本目前未支持处理地形和移动.
+extra.terrainandmovement_required=请先在配置文件中启用地形和移动.
+# Inventory
+extra.inventory_enabled=库存处理现在已启用.
+extra.inventory_disabled=§c该MC版本目前未支持处理库存.
+extra.inventory_required=请先在配置文件中启用库存处理.
+extra.inventory_interact=使用 /inventory 来与其交互.
+extra.inventory_open=Inventory # {0} opened: {1}
```
Comment (ReinforceZwei) [inline MinecraftClient/Resources/lang/zh-CHS.ini:136]: Missing translation here.

**Review by ReinforceZwei — COMMENTED** [review]
Hi, thank you for your contribution. Before merging your PR, there's some feedback for you.

**Comment by Cubik65536** [comment]
Thanks for the feedback! I fixed those problems and changed translations of  `inventory`, `slot` and `stack` to those from Minecraft Wiki

**Comment by ORelio** [comment]
Good thing we have @ReinforceZwei for reviewing your PR, I'm totally useless here (can't read Chinese 😜)

**Comment by Cubik65536** [comment]
> Good thing we have @ReinforceZwei for reviewing your PR, I'm totally useless here (can't read Chinese 😜)

It makes me feel good to be useful, but I have no idea how to develop a program with C#, so I'm just doing some localization work with my team😜

**Review by ReinforceZwei — APPROVED** [review]
Looks good 👍

**Pull request merged**